doc/source/getting-started/abbr.rst

```diff
@@ -138,6 +286,24 @@ Open source software may be developed in a collaborative public manner.
 P - Q - R
 ---------
 
+**PCBN**
+
+Printed circuit board: An assembly that mechanically supports and
+electrically connects layers of conductors, metal interconnects,
+insulators, and other components such as diodes and resistors.
+
+**PCBN**
+
+Polycrystalline Cubic Boron Nitride: A cutting tool used in the
+machining of extremely hard or abrasive materials. The `PyMAPDL example <https://examples.mapdl.docs.pyansys.com/technology_showcase_examples/techdemo-28/ex_28-tecfricstir.html>`_,
```

Comment:
Is it the same example as before?

Reply:
@RobPasMue It was--but because of the error generated, I revisited this definition and changed it to this:

Polycrystalline cubic boron nitride: A super-abrasive material that is used to
make cutting tool for machining extremely hard materials. This tool is used
in FSW (friction stir welding).